File: Common/Entity/src/main/java/it/ey/dto/StakeHolderDTO.java
package it.ey.dto;


import lombok.*;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class StakeHolderDTO extends CampiTecniciDTO {
    private Long id;
    private Long idPiao;
    private String nomeStakeHolder;
    private String relazionePA;
}

File: PIAO_FE/src/app/shared/models/classes/stakeholder-dto.ts
import { CampiTecniciDTO } from './campi-tecnici-dto';

export class StakeHolderDTO extends CampiTecniciDTO {
  id?: number;
  idPiao?: number;
  nomeStakeHolder?: string;
  relazionePA?: string;

  override toString(): string {
    return `${this.nomeStakeHolder} - ${this.relazionePA}`;
  }
}
